cmd/state-tray/main.go

```diff
@@ -57,28 +72,56 @@ func run() error {
 
 	systray.AddSeparator()
 
+	mProjects := systray.AddMenuItem(locale.Tl("tray_projects_title", "Local Projects"), "")
+	mReload := mProjects.AddSubMenuItem("Reload", "Reload the local projects listing")
+	localProjectsUpdater := menu.NewLocalProjectsUpdater(mProjects)
```

Comment:
I'm not sure about this approach. I think it would be better to wrap the `systray` library and return our own menu items that can handle specialized behavior. Something like:

```
mProjects := menu.NewLocalProjectsMenu(localProjects)
...
case <-mReload.ClickedCh:
	logging.Debug("Projects event")
	localProjects, err = model.LocalProjects()
	if err != nil {
		logging.Error("Could not get local projects listing: %v", err)
	}
	mProjects.Reload()
```

This could be done in this PR but would lead to different ways of creating a new menu item. Also, wrapping `systray` felt out of scope. I've created a ticket here: https://www.pivotaltracker.com/story/show/177551480